db/database.go

```diff
@@ -61,6 +61,13 @@ var (
 	DefaultDeltaSyncRevMaxAge = uint32(60 * 60 * 24) // 24 hours in seconds
 )
 
+var DefaultCompactInterval = uint32(1440) // Default compact interval in minutes
+
+const (
+	CompactIntervalMin = 60
+	CompactIntervalMax = 86400
+)
```

Comment:
Would be nice to break out into the 60 * 24 * 60 format to make it clearer what the value corresponds to.

Reply:
Would also be nice to add a suffix to indicate minutes (or seconds).